var/spack/repos/builtin/packages/drishti/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Drishti(PythonPackage):
+    """
+    Drishti is a command-line tool to guide end-users in optimizing I/O in their applications
+    by detecting typical I/O performance pitfalls and providing a set of recommendations.
+    """
+
+    homepage = "https://github.com/hpc-io/drishti"
+    git = "https://github.com/hpc-io/drishti"
+    pypi = "drishti-io/drishti-io-0.4.tar.gz"
+
+    maintainers = ["jeanbez", "sbyna"]
+
+    version("latest", branch="master")
```

Comment:
Why not use `master` as the name of the version?  It is "standard" and would be clearer to people wanting to build based on the branch.

Reply:
Confirmed the branch exists.